src/cli/commands/install.js

```diff
@@ -283,13 +284,6 @@ export class Install {
       Object.assign(this.resolutions, projectManifestJson.resolutions);
       Object.assign(manifest, projectManifestJson);
 
-      this.resolutionMap.init(this.resolutions);
```

Comment:
This is moved to after `pushDeps` is so that we know what dependencies to not add